File: server/order-service/src/main/java/com/sudo248/orderservice/controller/payment/dto/PaymentDto.java
package com.sudo248.orderservice.controller.payment.dto;

import com.sudo248.orderservice.repository.entity.payment.PaymentStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.TimeZone;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaymentDto {
    private String paymentId;
    private String orderId;
    private String orderType;
    private String bankCode;
    private Double amount;
    private String paymentType;
    private String ipAddress;
    private String timeZoneId;
    private String paymentUrl;
    private PaymentStatus paymentStatus;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/payment/PaymentDto.kt
package com.sudo248.sudoo.data.dto.payment

data class PaymentDto(
    val paymentId: String = "",
    val paymentUrl: String? = null,
    val paymentType: String,
    val bankCode: String? = null,
    val amount: Double = 0.0,
    val orderId: String,
    val orderType: String = "100000",
    val ipAddress: String? = null,
    val paymentStatus: String
)
